wazo_dird_client/client.py

```diff
@@ -2,10 +2,29 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from wazo_lib_rest_client.client import BaseClient
+import wazo_dird_client.commands
 
 
 class DirdClient(BaseClient):
     namespace = 'wazo_dird_client.commands'
 
+    backends: wazo_dird_client.commands.backends.BackendsCommand
+    conference_source: wazo_dird_client.commands.conference_source.Command
+    config: wazo_dird_client.commands.config.ConfigCommand
+    csv_source: wazo_dird_client.commands.csv_source.Command
+    csv_ws_source: wazo_dird_client.commands.csv_ws_source.Command
+    directories: wazo_dird_client.commands.directories.DirectoriesCommand
+    displays: wazo_dird_client.commands.displays.DisplaysCommand
+    graphql: wazo_dird_client.commands.graphql.GraphQLCommand
+    ldap_source: wazo_dird_client.commands.ldap_source.Command
+    personal: wazo_dird_client.commands.personal.PersonalCommand
+    phonebook: wazo_dird_client.commands.phonebook.PhonebookCommand
```

Comment:
phonebook_deprecated should be added here and in setup.py to be usable or be removed